packages/contentstack-config/src/commands/config/set/rate-limit.ts

```diff
@@ -0,0 +1,80 @@
+import { flags, isAuthenticated, FlagInput, managementSDKClient, cliux } from '@contentstack/cli-utilities';
+import { RateLimitHandler } from '../../../utils/rate-limit-handler';
+import { BaseCommand } from '../../../base-command';
+import { askOrgID } from '../../../utils/interactive';
+
+interface RateLimitConfig {
+  org?: string;
+  utilize?: number;
+  limitName?: string[];
+  default?: boolean;
+  auth_token?: string;
+}
+
+export default class RateLimitSetCommand extends BaseCommand<typeof RateLimitSetCommand> {
+  static description = 'Set rate-limit for CLI';
+
+  static flags: FlagInput = {
+    org: flags.string({
+      description: 'Provide the organization UID',
+    }),
+
+    utilize: flags.string({
+      description: 'Provide the utilization percentage',
+      default: '50',
+    }),
+
+    'limit-name': flags.string({
+      description: '[Optional] Provide the limit names separated by comma',
+      multiple: true,
+    }),
```

Comment:
@harshithad0703 ,
Provide option here similar like. 
 https://github.com/contentstack/cli/pull/1544/files#diff-59a0e4886e4b8e0671ef051c352241f147b2c3324b7a2a6cef3e9d0a68eb4d9dR27-R30
 
In options you can provide ['limit', 'getLimit', 'bulkLimit']. Keep it in some file so that we can use it everywhere. Otherwise user can provide any name